packages/playground/src/weblets/profile_manager.vue

```diff
@@ -486,6 +485,7 @@ async function mounted() {
     if (grid) {
       const DepositFee = await grid.bridge.getDepositFee();
       depositFee.value = DepositFee;
+      return;
```

Comment:
I'm curious to know why did you returned here?; there is nothing after.

Reply:
I think this was to prevent it from reloading, before adding this return it used reload and close the profile manage if you opened it by clicking on the balance section.